### ISSUE TITLE
AttributeError: 'MultiAgentDialogWorld' object has no attribute 'acts'

### ISSUE THREAD

**Issue by jojonki** [facebookresearch/ParlAI#497]
I want to use three agents. So the world is MultiAgentDialogWorld. But I got the following error when I set `batch_size`. I prepared a very simple for reproducing. Please see a below diff.

### diff
```
--- a/examples/train_model.py
+++ b/examples/train_model.py
@@ -122,7 +122,7 @@ class TrainLoop():
             build_dict(opt)
         # Create model and assign it to the specified task
         self.agent = create_agent(opt)
-        self.world = create_task(opt, self.agent)
+        self.world = create_task(opt, [self.agent, create_agent(opt)])
```

### Result
```
$ python examples/train_model.py -t dialog_babi:Task:1 -m memnn -bs 2
...
Traceback (most recent call last):
  File "examples/train_model.py", line 237, in <module>
    TrainLoop(setup_args()).train()
  File "examples/train_model.py", line 201, in train
    world.parley()
  File "/home/jonki/work/ParlAI/parlai/core/worlds.py", line 654, in parley
    batch_act = self.batch_act(agent_idx, batch_observations[agent_idx])
  File "/home/jonki/work/ParlAI/parlai/core/worlds.py", line 629, in batch_act
    acts = w.get_acts()
  File "/home/jonki/work/ParlAI/parlai/core/worlds.py", line 185, in get_acts
    return self.acts
AttributeError: 'MultiAgentDialogWorld' object has no attribute 'acts'
```

**Comment by klshuster** [comment]
Hi, I was not able to reproduce the error you had; could you please try pulling the latest code and running it again to see if it still does not work for you?

**Comment by alexholdenmiller** [comment]
was able to repro with a slightly different command, fix out #505
